src/instruments/src/PFD/FMA.tsx

```diff
@@ -1,567 +1,1178 @@
+                this.displayModeChangedPath();
+                break;
+            case 8:
+                text = '<text  class="FontMedium MiddleAlign Green" x="16.782249" y="7.1280665">MACH</text>';
+                this.displayModeChangedPath();
+                break;
+            case 9:
+                text = '<text  class="FontMedium MiddleAlign Green" x="16.782249" y="7.1280665">THR MCT</text>';
+                this.displayModeChangedPath();
+                break;
+            case 10:
+                text = '<text  class="FontMedium MiddleAlign Green" x="16.782249" y="7.1280665">THR CLB</text>';
+                this.displayModeChangedPath();
+                break;
+            case 11:
+                text = '<text  class="FontMedium MiddleAlign Green" x="16.782249" y="7.1280665">THR LVR</text>';
+                this.displayModeChangedPath();
+                break;
+            case 12:
+                text = '<text  class="FontMedium MiddleAlign Green" x="16.782249" y="7.1280665">THR IDLE</text>';
+                this.displayModeChangedPath();
+                break;
+            case 13:
+                text = `<g>
+                                <path class="NormalStroke Amber BlinkInfinite" d="m0.70556 1.8143h30.927v6.0476h-30.927z" />
+                                <text class="FontMedium MiddleAlign Green" x="16.782249" y="7.1280665">A.FLOOR</text>
+                            </g>`;
+                break;
+            case 14:
+                text = `<g>
+                                <path class="NormalStroke Amber BlinkInfinite" d="m0.70556 1.8143h30.927v6.0476h-30.927z" />
+                                <text class="FontMedium MiddleAlign Green" x="16.782249" y="7.1280665">TOGA LK</text>
+                            </g>`;
+                break;
+            default:
+                text = '';
+                this.isShown = false;
+            }
+        }
 
-const A3Cell = ({ ABrkMode, ABrkActive, AB3Message }) => {
-    const engineMessage = getSimVar('L:A32NX_AUTOTHRUST_MODE_MESSAGE', 'enum');
+        this.cellRef.instance.innerHTML = text;
+    }
 
-    let text: string;
-    let className: string;
-    if (engineMessage === 1) {
-        text = 'THR LK';
-        className = 'Amber BlinkInfinite';
-    } else if (engineMessage === 2) {
-        text = 'LVR TOGA';
-        className = 'White BlinkInfinite';
-    } else if (engineMessage === 3) {
-        text = 'LVR CLB';
-        className = 'White BlinkInfinite';
-    } else if (engineMessage === 4) {
-        text = 'LVR MCT';
-        className = 'White BlinkInfinite';
-    } else if (engineMessage === 5) {
-        text = 'LVR ASYM';
-        className = 'Amber';
-    } else if (ABrkMode === 3 && !ABrkActive && !AB3Message) {
-        text = 'BRK MAX';
-        className = 'Cyan';
-    } else {
-        return null;
+    private setAutoBrakeText(newAutobrakeActive: boolean) {
```

Comment:
The A/BRK and A/THR text setting should not be seperate like that.
If I'm not mistaken, it is theoretically possible now that an A/THR mode is still active, while the A/BRK active mode is shown. (Yes, this is a very remote possibility, but nevertheless it should be considered)